ScanCommand/src/test/java/io/ballerina/scan/Main.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Check the indentation here with other existing files

Reply:
Fixed in [5556927](https://github.com/ballerina-platform/static-code-analysis-tool/pull/2/commits/55569279cd21bc02fdae8dc9652f2628edfccf48)